packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -371,9 +371,19 @@ export class MLSService extends TypedEventEmitter<Events> {
     return response;
   }
 
-  public removeClientsFromConversation(conversationId: ConversationId, clientIds: Uint8Array[]) {
-    return this.processCommitAction(conversationId, () =>
-      this.coreCryptoClient.removeClientsFromConversation(conversationId, clientIds),
+  /**
+   * Will send a commit with removal proposal for given clients
```

Comment:
I'm not entirely sure it sends a proposal, I believe it a commit, isn't it?

Reply:
I'm not sure either 😅 Let's call it removal commit 👍